elide-core/src/main/java/com/yahoo/elide/core/filter/FilterPredicate.java

```diff
@@ -199,7 +199,6 @@ public String getPlaceholder() {
         }
 
         public String escapeMatching() {
-            // It is unclear why replaceAll here breaks our tests
```

Comment:
FYI: `replaceAll` is not a replacement for `replace`.  If the target is a literal always use `replace`.